packages/components/app/styles/components/side-nav.scss

```diff
@@ -0,0 +1,142 @@
+//
+// SIDE-NAV
+//
+
+// WIP, naming, etc. will be refined...
+:root {
+  --nav-item-padding-horizontal: 12px;
+}
+
+
+// * SideNav Parent (wrapper) component
+// -------------------------------------------------------------------
+
+.hds-side-nav {
+  // NOTE: Cloud-ui uses grid layout in non-mobile
+  position: fixed; // TODO: should apply only to "mobile" view
```

Comment:
TBD if we should take care of this, or it's their concern to implement the positioning and responsiveness of the component (probably it's ours, but that's not 100% sure yet).
That's why I suggested to start from low-level components (atoms) and worry about the top-level positioning and responsiveness only later. It's easier to solve smaller problems, to gain knowledge on the domain/problem space, and use it later to solve bigger problems.

Reply:
I commented the positioning out for now. Will probably delete before release but may need to use the positioning to test things like overflowing content.